core/src/test/java/com/whylogs/core/ColumnProfileTest.java

```diff
@@ -25,6 +26,27 @@ public void column_BasicTracking_ShouldWork() {
     assertThat(col.getNumberTracker().getDoubles().getCount(), is(2L));
   }
 
+  /** Check that custom null specification detects nulls and has no false positives. */
+  @Test
+  public void column_NullTest_ShouldWork() {
+    val col = new ColumnProfile("test", ImmutableList.copyOf("nil.NaN,nan,null".split(",")));
```

Comment:
```suggestion
    val col = new ColumnProfile("test", ImmutableList.of("nil", "NaN","nan","null"));
```

Here you're not testing `String.split` so it's better to make the test input really explicit